x-pack/solutions/observability/plugins/streams_app/public/components/stream_detail_enrichment/flyout/ignore_toggles.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { useWatch } from 'react-hook-form';
+import { EuiCode, EuiText } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { ToggleField } from './toggle_field';
+
+export const IgnoreFailureToggle = () => {
+  const value = useWatch({ name: 'ignore_failure' });
+
+  return (
+    <ToggleField
+      name="ignore_failure"
+      label={i18n.translate(
+        'xpack.streams.streamDetailView.managementTab.enrichment.processorFlyout.ignoreFailuresLabel',
+        { defaultMessage: 'Ignore failures for this processor' }
+      )}
+      helpText={
+        !value ? (
+          <EuiText component="span" size="relative" color="warning">
+            <FormattedMessage
+              id="xpack.streams.streamDetailView.managementTab.enrichment.processorFlyout.ignoreFailuresWarning"
+              defaultMessage="Disabling the {ignoreField} option could lead to unexpected pipeline failures."
+              values={{
+                ignoreField: <EuiCode>ignore_failure</EuiCode>,
+              }}
+            />
+          </EuiText>
+        ) : undefined
+      }
+    />
+  );
+};
+
+export const IgnoreMissingToggle = () => {
+  return (
+    <ToggleField
+      name="ignore_missing"
+      label={i18n.translate(
+        'xpack.streams.streamDetailView.managementTab.enrichment.processorFlyout.ignoreMissingLabel',
+        { defaultMessage: 'Ignore missing' }
```

Comment:
Nit: With the way labels get used in accessibility contexts should we go “Ignore missing fields” here too as it’s less ambiguous?

Reply:
I used the same copies+help text from the ingest pipeline management, here the thing is that the "Ignore missing" refers to ignore "documents" with the missing field... so maybe "Ignore missing fields" is also not accurate, wdyt? I guess the help caption here is used specifically for this precisation.

Reply:
Gotcha, thanks for the explanation. Let's just leave it as it's consistent with the pipeline management 👌